pom.xml

```
@@ -31,7 +31,7 @@
     <groupId>org.dependencytrack</groupId>
     <artifactId>dependency-track</artifactId>
     <packaging>war</packaging>
-    <version>5.0.2</version>
+    <version>5.0.3</version>
```

Should have the `-SNAPSHOT` suffix, the release workflow will automatically remove the suffix during release.

```suggestion
    <version>5.0.3-SNAPSHOT</version>
```